src/sentry/search/events/datasets/spans_metrics.py

```diff
@@ -805,6 +821,101 @@ def _resolve_rate(
             alias,
         )
 
+    def _resolve_regression_score(
+        self, args: Mapping[str, str | Column | SelectType | int | float], alias: str | None = None
+    ) -> SelectType:
+        return Function(
+            "minus",
+            [
+                Function(
+                    "multiply",
+                    [
+                        self._resolve_average_cond(args, None, "greater"),
+                        self._resolve_epm_cond(args, None, "greater"),
+                    ],
+                ),
+                Function(
+                    "multiply",
+                    [
+                        self._resolve_average_cond(args, None, "less"),
+                        self._resolve_epm_cond(args, None, "less"),
+                    ],
+                ),
+            ],
+            alias,
+        )
+
+    def _resolve_epm_cond(
+        self,
+        args: Mapping[str, str | Column | SelectType | int | float],
+        alias: str | None,
+        cond: str,
+    ) -> SelectType:
+        assert self.builder.params.start
+        assert self.builder.params.end
+        timestamp = cast(datetime, args["timestamp"])
+        if cond == "greater":
+            interval = (self.builder.params.end - timestamp).total_seconds()
```

Comment:
Do we need to do a Math.abs here to make sure that delta is positive since timestamp is user supplied?